### DISCUSSION TITLE
Images not displayed in page.

### DISCUSSION THREAD

**Discussion by TonyxSun** [thuliteio/doks#338] [Q&A] [answered]
Hello,
Recent markdown content that I added is not displaying embedded images through html. The problem seems to be that the images are not getting hosted. 

This is how Im attaching images in my .md files:
``` 
markdown content

<img src="img_6.png" title="title" alt="alt title" width="600"/>

More markdown content 
```
How it looks locally:

![CleanShot 2021-06-16 at 11 52 12](https://user-images.githubusercontent.com/68176295/122276225-6ed98780-ce99-11eb-8529-33d8d6b0ca3e.png)
The image file is not available: 

![CleanShot 2021-06-16 at 11 48 35](https://user-images.githubusercontent.com/68176295/122275925-1a360c80-ce99-11eb-987c-1db3ef8622a3.png)

Directory structure:

![CleanShot 2021-06-16 at 11 51 01](https://user-images.githubusercontent.com/68176295/122276012-3639ae00-ce99-11eb-9832-b8031c8957d6.png)

What might be the problem? Thanks! Let me know if you need for info.

**Comment by h-enk** [comment, marked as answer]
Hi @TonyxSun, try using one of the [image shortcodes](https://getdoks.org/docs/recipes/images/).

Or, place you're image in `./static/images/`, and reference it like so:

```md

<img src="/images/img_6.png" title="title" alt="alt title" width="600"/>
```

**Reply by wenyaoleoliu** [reply]
> Hi @TonyxSun, try using one of the [image shortcodes](https://getdoks.org/docs/recipes/images/).
> 
> Or, place you're image in `./static/images/`, and reference it like so:
> 
> ```lisp
> <img src="/images/img_6.png" title="title" alt="alt title" width="600"/>
> ```

Many themes support linking the images in static/images folder the format of

```
![images under static/image folder](/images/imagename.png)
```

But the Doks theme seems do not support. Is there a plan or a way to set that up??

**Reply by h-enk** [reply]
Thanks for pointing this out! Think so too, that Doks should support this: https://github.com/h-enk/doks/issues/759
